src/main/resources/META-INF/mods.toml

```diff
@@ -19,7 +21,6 @@ displayName="Server Pack Locator Utility mod" #mandatory
 credits="cpw" #optional
 # A text field displayed in the mod UI
 authors="cpw" #optional
-
 license="LGPL v3"
```

Comment:
I think you can remove the field here

Reply:
Left it there in case he decides to change it forge side. Should do no harm to include it